addons/beehave/nodes/beehave_debug.gd

```diff
@@ -0,0 +1,102 @@
+extends BeehaveNode
+
+class_name BeehaveDebug, "../icons/tree.svg"
+
+export(NodePath) var _beehave_tree
+var beehave_tree : BeehaveRoot setget set_beehave_tree
+
+export(Color) var tick_color := Color.whitesmoke
```

Comment:
We might want to use colours from the current editor theme rather than using custom colours (as default). Perhaps we could even introduce a new section in the project settings where users can tweak these colours themselves! Since this is an addon, it should be farely easy to extend the existing Godot UI with it.

Reply:
That's a good idea! My current version of the script has a `theme` variable for styling the tree, I'm not sure which variables to use for the colors though